src/components/application_manager/include/application_manager/commands/hmi/navi_is_ready_request.h

```diff
@@ -55,12 +56,17 @@ class NaviIsReadyRequest : public RequestToHMI {
   /**
    * @brief NaviIsReadyRequest class destructor
    **/
-  virtual ~NaviIsReadyRequest();
+  ~NaviIsReadyRequest() OVERRIDE;
```

Comment:
@vlantonov I use this for consistency, it does not carry the semantic load. I will remove this with next commit.